Sources/Scotty/Route.swift

```diff
@@ -8,8 +8,8 @@
 
 import Foundation
 
-public struct Route<RootViewController: UIViewController> {
-    public typealias Navigator = (_ rootViewController: RootViewController, _ options: [AnyHashable: Any]?) -> Bool
+public struct Route<Root> {
+    public typealias Navigator = (_ rootViewController: Root, _ options: [AnyHashable: Any]?) -> Bool
```

Comment:
Should the word `rootViewController` be changed to `Root` here?

Reply:
Definitely should - fixed now.

Reply:
Why am I not seeing this change in the "Files changed" section? Did the commit get pushed?

Reply:
Should be good to go now.